modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseQueryServiceWaitStrategy.java

```diff
@@ -0,0 +1,53 @@
+package org.testcontainers.couchbase;
+
+import com.couchbase.client.core.message.cluster.GetClusterConfigRequest;
+import com.couchbase.client.core.message.cluster.GetClusterConfigResponse;
+import com.couchbase.client.core.service.ServiceType;
+import com.couchbase.client.java.Bucket;
+import org.rnorth.ducttape.TimeoutException;
+import org.testcontainers.containers.ContainerLaunchException;
+import org.testcontainers.containers.GenericContainer;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+import static org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess;
+
+/**
+ * @author ctayeb
+ * Created on 06/06/2017
+ */
+public class CouchbaseQueryServiceWaitStrategy extends GenericContainer.AbstractWaitStrategy {
```

Comment:
We have a new base class not based on `GenericContainer.AbstractWaitStrategy`, could you please use it?